conductr_cli/conduct_info.py

```diff
@@ -17,7 +17,7 @@ def info(args):
 
     data = [
         {
-            'id': bundle['bundleId'] if args.long_ids else bundle_utils.short_id(bundle['bundleId']),
+            'id': ('! ' if bundle.get('hasError', False) else '') + (bundle['bundleId'] if args.long_ids else bundle_utils.short_id(bundle['bundleId'])),
```

Comment:
Is it possible to color the ID including the exclamation mark red in the case of an error?